src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -56,11 +67,27 @@ public function storeHasAProductPricedAt($productName, $currency, $price)
         $product = $this->productFactory->createNew();
         $product->setName($productName);
         $product->setPrice((int) $price * 100);
-        $product->setDescription('Awesome star wars mug');
+        $product->setDescription('Awesome '.$productName);
 
         $channel = $this->sharedStorage->getCurrentResource('channel');
         $product->addChannel($channel);
 
         $this->productRepository->add($product);
     }
+
+    /**
+     * @Given /^product "([^"]*)" belongs to "([^"]*)" tax category$/
```

Comment:
Can we use transformers here? It will delegate the logic for retrieving resources:

```php
/**
 * @Given product :product belongs to :taxCategory tax category
 */
public function productBelongsToTaxCategory(ProductInterface $product, TaxCategoryInterface $taxCategory)
{
    $product->setTaxCategory($taxCategory);
}
```

Also, let's use `+` instead of `*` to disallow empty names.